crates/bevy_gltf/Cargo.toml

```diff
@@ -29,4 +29,4 @@ thiserror = "1.0"
 anyhow = "1.0.4"
 base64 = "0.13.0"
 percent-encoding = "2.1"
-wgpu = "0.11.0"
+wgpu = { version = "0.12.0" }
```

Comment:
when theres no other configuration we favor the simpler `wgpu = "0.12.0"` format

Reply:
fixed